packages/next/src/build/index.ts

```diff
@@ -3220,6 +3249,7 @@ export default async function build(
                   srcRoute: page,
                   dataRoute,
                   prefetchDataRoute,
+                  allowHeader: ALLOWED_HEADERS,
```

Comment:
Should we only put these on `srcRoute` entries to avoid inflating the manifest too much?

Reply:
They also apply to revalidations on routes that aren't dynamic, so it has to be both.